.github/workflows/other-lint.yml

```diff
@@ -27,11 +27,15 @@ jobs:
           VALIDATE_PYTHON: false
           VALIDATE_JAVASCRIPT_ES: false
           VALIDATE_JAVASCRIPT_STANDARD: false
+          VALIDATE_TYPESCRIPT_ES: false
+          VALIDATE_TYPESCRIPT_STANDARD: false
           VALIDATE_PYTHON_BLACK: false
           VALIDATE_PYTHON_FLAKE8: false
           VALIDATE_PYTHON_ISORT: false
           VALIDATE_PYTHON_MYPY: false
           VALIDATE_PYTHON_PYLINT: false
           VALIDATE_GO: false
           VALIDATE_JSCPD: false
-          FILTER_REGEX_EXCLUDE: (java/gradlew)
\ No newline at end of file
+          VALIDATE_PHP_PHPSTAN: false
+          VALIDATE_PHP_PSALM: false
```

Comment:
I disabled these because they are not able to find the dependencies (I think they aren't running `composer` properly before trying to run static analysis, we still have PHP CodeSniffer enabled, PHP STAN passes locally but I have to break PHP out into a separate linter that will run composer as a preflight, I'll do this in a separate PR.

Reply:
This "superlinter" isn't proving itself, haha. :)

Ready to go in?

Reply:
Yeah, I was hoping it would remove a step when adding a new language but it ended up being more work than it's worth, will likely replace this with individual workflows for each language soon.

If everything looks good to you I'd say it's ready to be merged.